app/views/fragments/javascriptFirstSteps.scala.html

```diff
@@ -96,3 +96,18 @@
         }
     })(guardian.isModernBrowser);
 </script>
+
+
+<script id="gu_page_info">
+    guardian.pageInfo = {
```

Comment:
Minor niggle, but personally I'd find it clearer to define this next to the other `guardian.` definitions in `id="gu"`. We could always move the `isModernBrowser` scoped code into another script block if that aids clarity. If that made any sense.